index.js

```diff
@@ -595,6 +599,38 @@ class XiaomiRoborockVacuum {
     }
   }
 
+  async getRoomList() {
+    await this.ensureDevice('getRoomList');
+
+    try {
+      const timers = await this.device.call('get_timer');
+
+      // Find specific timer containing the room order
+      // Timer needs to be scheduled for 00:00 and inactive
+      let leetTimer = timers.find(
+        x => x[2][0].startsWith("0 0") && x[1] == 'off');
+      if (leetTimer == undefined) {
+        this.log.error(`ERR getRoomList | ${this.model} | Could not find a timer for autoroom`);
+        return;
+      }
+
+      let roomIds = leetTimer[2][1][1]['segments'].split`,`.map(x=>+x);
```

Comment:
Are we missing a parenthesis here?
```suggestion
      let roomIds = leetTimer[2][1][1]['segments'].split(`,`).map(x=>+x);
```

Reply:
Stackoverflow said they could be omitted, but yeah it make more sense with them